docker/base-images.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+images:
+- gcr.io/fuzzbench/base-image:test-experiment
+- gcr.io/fuzzbench/base-image
+steps:
+- args:
+  - build
+  - --tag
+  - gcr.io/fuzzbench/base-image
+  - --tag
+  - gcr.io/fuzzbench/base-image:test-experiment
+  - --cache-from
+  - gcr.io/fuzzbench/base-image
+  - --build-arg
+  - BUILDKIT_INLINE_CACHE=1
+  - --file
+  - docker/base-image/Dockerfile
+  - docker/base-image
+  env: DOCKER_BUILDKIT=1
+  id: base-image
+  name: gcr.io/cloud-builders/docker
+  wait_for: []
```

Comment:
this wait_for seems unnecessary, consider removing it

Reply:
Noted, I'll remove this here. But the generation does add empty [] if nothing is there. I removed the delete empty list based on previous review comments, so if this is generated again, it would have to be manually removed.